### ISSUE TITLE
[maps/burger_shot] ATM in BurgerShot

### ISSUE THREAD

**Issue by Infamously1** [InZidiuZ/legacy-assets-issue-tracker#495]
**Name:**
Infamously#7170

**Summary:**
I recently acquired the BurgerShot franchise, when opening the doors the first thing I had noticed is that there was no ATM. How are people going to withdraw cash if there is no ATM, if an ATM is not put in BurgerShot San Andreas location and people must go all the way to the convenience store, 5 blocks away, they may as well just purchase the food at said convenience which is another issue I will talk about later on as Mr. Chambers advised me to wait until everyone see's that BurgerShot is a successful business. I hope this suggestion isn't too difficult to introduce, however without it, BurgerShot will face 

**Link:**
If there are any links to what you're suggesting, please add them here.
https://gyazo.com/dc1e941f666a12318b93516a5338f038
An image where I feel the ATM would be ideal.
**Reason:**
Why would your suggestion be a good addition to Legacy?
My suggestion is a good addition to Legacy because once I was chosen to take over BurgerShot, I made it my goal to remodel the whole philosophy on how the old BurgerShot operated, this is only the beginning, as stated before once BurgerShot is proven to be a thriving business I will make another suggestion once again. Nevertheless, the addition of this ATM is simply but crucial to the success of BurgerShot's future for the reasons stated in the summary. Hope you guys will consider my suggestion. Thank you for your time.

**Comment by kswagz** [comment]
Agreed. Too many times where I couldn't pay for something and had to awkwardly leave for a few minutes and come back.

The blue arrow that lets you buy food also makes it hard to RP it out and should be placed in the kitchen so that customers can't  just buy the food themselves and leave.

**Comment by XwezioGH** [comment]
This has been added!

**Xwezio-
Asset Team**

**Comment by antihero6713** [comment]
There has not been a update to assets pushed to the main server be please be patient. You are not the only one waiting on updates to stuff and things.

**Comment by Infamously1** [comment]
Oh alright, sorry about that, just heard EDM rotated so I thought ATM woulda been put in all good 👍
